### ISSUE TITLE
createVisualShape with GEOM_CYLINDER: mixup of keywords length and height

### ISSUE THREAD

**Issue by thomasvandewiel-sai** [bulletphysics/bullet3#4223]
When creating a cylindrical shape in Pybullet, the following function calls is suggested in the documentation:
`pybullet.createVisualShape(pybullet.GEOM_CYLINDER, radius=val, height=val)`.
This is consistent with the method for creating the collision shape for a cylindrical object.
`pybullet.createCollisionShape(pybullet.GEOM_CYLINDER, radius=val, height=val)`.

However, createVisualShape invokes the following error: 
`TypeError: 'height' is an invalid keyword argument for this function`

When changing the height keyword to length, this problem is solved, i.e. 
`pybullet.createVisualShape(pybullet.GEOM_CYLINDER, radius=val, length=val)` executes without errors.

In other words, the documentation proposes incorrect keyword argument usage, and the implementation uses an inconsistent keyword naming convention.

**Comment by erwincoumans** [comment]
The documentation for createVisualShape mentions length, not height for GEOM_CYLINDER
See page 14 of https://docs.google.com/document/d/10sXEhzFRSnvFcl3XxNGhnD4N2SedqwdAvK3dsihxVUA/edit#heading=h.2ye70wns7io3

<img width="766" alt="image" src="https://user-images.githubusercontent.com/725468/163311604-f9d9b5a5-7fa8-4f0a-94fa-9ce453163e38.png">

Yes, it is an annoying inconsistency in the implementation, so we just have to mention it in the documentation.

If above doesn't answer the question, where in the documentation does it suggest using height for pybullet.createVisualShape?
